src/soilbiogeochem/SoilBiogeochemCarbonFluxType.F90

```diff
@@ -58,13 +58,8 @@ module SoilBiogeochemCarbonFluxType
      real(r8), pointer :: lithr_col                                 (:)     ! (gC/m2/s) litter heterotrophic respiration: donor-pool based definition
      real(r8), pointer :: somhr_col                                 (:)     ! (gC/m2/s) soil organic matter heterotrophic res: donor-pool based definition
      real(r8), pointer :: soilc_change_col                          (:)     ! (gC/m2/s) FUN used soil C
-
-     ! fluxes to receive carbon inputs from FATES
-     real(r8), pointer :: FATES_c_to_litr_c_col                     (:,:,:) ! total litter coming from ED. gC/m3/s
```

Comment:
why is this flux three dimensional, is it one for each type of litter flux (lab, cel, lig).  If so, is this information somewhat redundant if the component fluxes are already included?

Reply:
Note this array is being totally removed from the code. So the question is moot.